src/app/games/game-summaries/game-summaries.component.html

```diff
@@ -1,3 +1,9 @@
+
+<h2>Games covered here</h2>
 <div fxLayout="row wrap" fxLayoutAlign="center">
   <game-summary fxFlex="25%" fxFlex.xs="100%" fxFlex.sm="33%" *ngFor="let game of games" [data]="game"></game-summary>
 </div>
+<h2>Games with their own website</h2>
+<div fxLayout="row wrap" fxLayoutAlign="center">
+  <external-game fxFlex="25%" fxFlex.xs="100%" fxFlex.sm="33%" *ngFor="let game of externalGames" [data]="game"></external-game>
```

Comment:
With only one game, do we need the for loop? I suppose if other sites wanted to make their own this would future proof things?

Reply:
It's probably simpler to do it that way than hardcode it in some way, I would say.